util/declarations/generator/index.mjs

```diff
@@ -116,13 +116,13 @@ const generateListeners = (json, schemas = {}) => compose(
   * Listen to all ${getModuleName(json)} events.
   * @param {Function} listener The listener function to handle the events.
   */
-  function listen(listener: (event: string, data: object) => {})
+  function listen(listener: (event: string, data: object) => void): Promise<bigint>
```

Comment:
Seems like there were two problems here:

1. Callback is required to return an empty object `{}` or `null`.
2. The return type of `listen` wasn't specified at all and is now `Promise<bigint>`.

Is that right?